application/app.py

```diff
@@ -398,6 +399,11 @@ def open_project(self):
         else:
             pass  # If no folder selected, don't load anything.
 
+    def open_feedback(self):
+        url = QtCore.QUrl('https://docs.google.com/forms/d/e/1FAIpQLSeTRwZlMUwNrbv9Nw-BddsOBGrCQjR5YXHbloPirRzB3-QoFA/viewform')
+        if not QtGui.QDesktopServices.openUrl(url):
+            QtGui.QMessageBox.warning(self, 'Connecting to Feedback', 'Could not open feedback form')
```

Comment:
Currently the warning box was untested as I could not trigger the false return state. This is the "accepted" method of targeting URL so I followed the convention.